internal/gps/vcs_source_test.go

```diff
@@ -15,14 +16,15 @@ import (
 	"testing"
 )
 
+var Strict *bool = flag.Bool("strict", false, "Fail tests when a required binary is missing, instead of skipping")
```

Comment:
Oops, I didn't intend to export that. It's 90% humidity in Chicago, my fingers are sweaty sausages. That's my excuse and I'm sticking (literally) to it.